src/lib/notificationHandlerWebsocket.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
```

Comment:
Logic would have to be a bit different actually. For robust notifications we would have to start a transaction on `transfers`, then post the message in the amqp queue and then close the transaction when successful. Websocket stuff is really light in comparison.